rpc/README.md

```diff
@@ -6936,6 +6936,8 @@ The JSON view of a transaction as well as its status.
 
 *   `cycles`: [`Cycle`](#type-cycle) `|` `null` - The transaction consumed cycles.
 
+*   `enter_tx_pool_timestamp`: [`Uint64`](#type-uint64) `|` `null` - If the transaction is in tx-pool, `enter_tx_pool_timestamp` represent when it enter the tx-pool
```

Comment:
I asked AI to give a better name:

Some better name options for enter_tx_pool_timestamp could be:
- tx_pool_entry_timestamp
- timestamp_entered_tx_pool
- tx_pool_inclusion_time
- time_added_to_tx_pool
These names are a bit more verbose but convey the meaning clearly that it is the timestamp when the transaction entered the transaction pool.

I think we can use `time_added_to_pool`. The prefix `tx_` is not necessary.